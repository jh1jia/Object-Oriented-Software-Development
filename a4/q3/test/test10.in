stdin count 2 dropfirst 1 doublewords doublewords count 1
teou123 onetuh1
