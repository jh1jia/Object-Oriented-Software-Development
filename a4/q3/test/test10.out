eou123
eou343
eou563
eou783
netuh9
netuh10
netuh11
netuh12
